kolibri/plugins/style_guide/assets/src/views/content/components/buttons.vue

```diff
@@ -200,8 +204,16 @@
 </style>
 `;
 
+  const api = require('!vue-doc!keen-ui/src/UiButton'); // eslint-disable-line
+
   module.exports = {
-    data: () => ({ codeExamplesTemplate })
+    components: {
+      'api-table': require('../api_table'),
+    },
+    data: () => ({ 
```

Comment:
Do you mean because I added "// eslint-disable-line" ? I added that because the html linter doesn't accept bang syntax in require statements. Let me look into whether there is a way to disable this, but I am not sure there is